File: C/Easy/2011.c
int finalValueAfterOperations(char **operations, int operationsSize)
{
    int answer = 0;

    char *word;
    for (int i = 0; i < operationsSize; i++)
    {
        word = operations[i];
        // x in front
        if (word[0] == 'X')
        {
            if (word[1] == '-')
            {
                answer--;
            }
            else
            {
                answer++;
            }
        }
        else
        { // x behind
            if (word[0] == '-')
            {
                answer--;
            }
            else
            {
                answer++;
            }
        }
    }

    return answer;
}

File: Java/Easy/2011.java
class Solution {
    public int finalValueAfterOperations(String[] operations) {
        int answer = 0;

        for (String temp : operations) {
            if (temp.charAt(0) == 'X') {
                if (temp.charAt(1) == '-') {
                    answer--;
                } else {
                    answer++;
                }
            } else {
                if (temp.charAt(0) == '-') {
                    answer--;
                } else {
                    answer++;
                }
            }
        }

        return answer;
    }
}